Readme.org

```diff
@@ -6,89 +6,89 @@ computations are written in Chicken.
 
 And Chicken is [[https://call-cc.org][R5RS Scheme]].
 
-* Compilation and usage
+* Compilation
 First, get yourself a Chicken. Second, some Chicken deps.
 
 ** Install Chicken Eggs
 By the way, there's a way to make Chicken install eggs non-globally. Refer to
 the documentation for more info:
 https://wiki.call-cc.org/man/5/Extension%20tools#changing-the-repository-location
 
-~chicken-install fmt getopt-long brev-separate~
+~chicken-install fmt getopt-long brev-separate test~
```

Comment:
```suggestion
~chicken-install fmt getopt-long brev-separate tree srfi-1 srfi-13 test~
```

I suppose we should use a file like requirements.txt or something and write here something like `use chicken-install < requirements.txt` instead of this forever growing thing just imagine 15 more libraries here huh

Reply:
I agree.